cves/CVE-2012-4386.yml

```diff
@@ -205,4 +237,11 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The thing that stuck out to me about this vulnerability is that the attack
+    vector does not come from a user parameter. The attack occurs when a user
+    takes a value set by the developer (the name of a URL query parameter) and
+    hijacks it, setting it to a new, specially-crafted value which creates a
+    Cross-Site Request Forgery. It is important to note that even though the
+    developer chooses query parameter names, these can be modified with a few simple
+    keystrokes in a browser's address bar, and so should also be validated.
```

Comment:
Really minor thing, but the file should end with a newline (hence that little indicator there that says "no newline at end of file")